webapp/utils/text_formatting.jsx

```diff
@@ -166,8 +166,13 @@ export function autolinkAtMentions(text, tokens, usernameMap) {
         const index = tokens.size;
         const alias = `$MM_ATMENTION${index}`;
 
+        let tokenValue = `<span data-mention='${username}'><a class='mention-link' href='#'>${mention}</a></span>`;
+        if (Constants.SPECIAL_MENTIONS.indexOf(username) >= 0) {
```

Comment:
Regarding [this](https://github.com/mattermost/platform/pull/6129#discussion_r114024225), this code isn't actually used for highlighting mentions, only for turning them into links. The highlighting part is done in `highlightCurrentMentions`. So right now, the code in `mentionExists` and `addToken` for special mentions doesn't really do anything. I'll file a ticket so we can deal with that later though. I'd like to get this in and running on pre-release a few days before we go to cut the release